pmlc/dialect/stdx/tests/i1StorageToi8.mlir

```diff
@@ -0,0 +1,30 @@
+// RUN: pmlc-opt -stdx-i1-storage-to-i8 %s | FileCheck %s
+
+module {
```

Comment:
nit: We usually don't need the `module` in a `.mlir` test (but it doesn't hurt either).

Reply:
I will remove the module part if it is redundant. Seems like there is a bit inconsistency with using this, other mlir tests from this dialect has it, also pxa have those, and for example tile does not.